org.eclipse.m2e.jdt.tests/projects/compilerEnablePreviewSettings/pom.xml

```
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>foo.bar</groupId>
+	<artifactId>compilerEnablePreviewSettings</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<properties>
+	        <maven.compiler.enablePreview>false</maven.compiler.enablePreview>
+	</properties>
+	<build>
+		<pluginManagement>
+			<plugins>	
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-compiler-plugin</artifactId>
+					<version>3.8.1</version>
+					<configuration>
+						<release>10</release>
+					</configuration>
+				</plugin>
+			</plugins>
+		</pluginManagement>
+	</build>

```

It is probably sufficient to specifiy the compiler.release as property and rely on the default binding for the maven-compiler plugin (especially its version).
```suggestion
	<properties>
	        <maven.compiler.enablePreview>false</maven.compiler.enablePreview>
	        <maven.compiler.release>11</maven.compiler.release>
	</properties>
```